test/hotspot/jtreg/compiler/jvmci/jdk.vm.ci.hotspot.test/src/jdk/vm/ci/hotspot/test/HotSpotResolvedJavaFieldTest.java

```diff
@@ -61,7 +69,7 @@ public class HotSpotResolvedJavaFieldTest {
         Field f = null;
         try {
             Class<?> typeImpl = Class.forName("jdk.vm.ci.hotspot.HotSpotResolvedObjectTypeImpl");
-            m = typeImpl.getDeclaredMethod("createField", JavaType.class, long.class, int.class, int.class);
+            m = typeImpl.getDeclaredMethod("createField", JavaType.class, int.class, int.class, int.class);
```

Comment:
Note this is needed as the mentioned method in class `HotSpotResolvedObjectTypeImpl` has `int` signature for the second param. Not `long`.